cli/lib/tasks/verify.js

```diff
@@ -191,37 +190,69 @@ const start = (options = {}) => {
 
   const packageVersion = util.pkgVersion()
   let binaryDir = state.getBinaryDir(packageVersion)
-  if (process.env.CYPRESS_BINARY_FOLDER) {
-    const envBinaryDir = process.env.CYPRESS_BINARY_FOLDER
-    logger.warn(stripIndent`
-    ${logSymbols.warning} Warning: You have set the environment variable: ${chalk.white('CYPRESS_BINARY_FOLDER=')}${chalk.cyan(envBinaryDir)}
-    
-      This overrides the default Cypress binary version used.
-    `)
-    logger.log()
-    binaryDir = envBinaryDir
-  }
 
   _.defaults(options, {
     force: false,
     welcomeMessage: true,
   })
 
-  return isMissingExecutable(binaryDir)
+  const checkEnvVar = () => {
+    debug('checking env variables')
+    if (process.env.CYPRESS_RUN_BINARY) {
+      const envBinaryPath = process.env.CYPRESS_RUN_BINARY
+      debug('CYPRESS_RUN_BINARY exists, =', envBinaryPath)
+      logger.warn(stripIndent`
+        ${logSymbols.warning} Warning: You have set the environment variable: ${chalk.white('CYPRESS_RUN_BINARY=')}${chalk.cyan(envBinaryPath)}
```

Comment:
Should this say "Warning"? It's likely what they intended, yet the printout seems to indicate there's something wrong with this with the 'Warning' messaging.

Reply:
We similarly warn for global install, which is also probably intentional...It's just possible that a user exports the env var and has no clue what's happening...but i'd be ok with not having "Warning". Maybe change it to "Note:" ?

Reply:
Yeah, I think that'd be better 👍